client/app/widgets/Notifications/__tests__/Notifications.spec.jsx

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React from 'react';
+import axios from 'axios';
+import { mount } from 'enzyme';
+import { Notifications } from '../index';
+
+let axiosGetSpy;
+let axiosDeleteSpy;
+let componentDidMountSpy;
+
+const data = {
+  signed_in: 1,
+  fetch_notifications: ['Notification 1'],
+};
+
+const component = (
+  <Notifications
+    element={<button type="button">Notifications</button>}
+    plural="Notifications"
```

Comment:
Ooh, this is an opportunity to refactor this component so we don't need to pass in prop. We were originally doing this because i18n wasn't working in the React side. 

You can pull in the translation key directly in the component by importing the i18n service from the libs directory and using `I18n.t('notifications.plural')` in the component.